resp/resp.go

```diff
@@ -24,7 +26,7 @@ func LookupFS(fsys fs.FS, response string) ([]byte, error) {
 
 		// return the first one found
 		for _, name := range names {
-			data, err := fs.ReadFile(files, name)
+			data, err := fs.ReadFile(fsys, name)
```

Comment:
question: don't quite understand this, is this a bugfix?

Reply:
Ah, yes, realised after testing that the variable was being shadowed.